helm-chart/values.yaml

```diff
@@ -17,9 +17,8 @@ tap:
     host: 127.0.0.1
   regex: .*
   namespaces: []
-  selfnamespace: kubeshark
   storagelimit: 200Mi
-  storageclass: standard
+  storageclass: ""
```

Comment:
This file is also generated automatically and shouldn't be modified directly. The `values.yaml` should match the output of `kubeshark config` command which prints the `config.yaml`. (prints a default structure if `$HOME/.kubeshark/config.yaml` is not present)

`values.yaml` is automatically generated on these lines: https://github.com/kubeshark/kubeshark/blob/2563cc19221fb5007411613694777c0e80b717c1/cmd/helmChart.go#L350-L360